packages/react-composites/src/composites/common/PeoplePaneContent.tsx

```diff
@@ -95,26 +107,21 @@ export const PeoplePaneContent = (props: {
   const theme = useTheme();
 
   const copyLinkButtonStylesThemed = useMemo(
-    () =>
-      concatStyleSets(copyLinkButtonStyles, {
-        root: {
-          minHeight: props.mobileView ? '3rem' : '2.5rem',
-          borderRadius: props.mobileView ? theme.effects.roundedCorner6 : theme.effects.roundedCorner4
-        }
-      }),
-    [props.mobileView, theme.effects.roundedCorner6, theme.effects.roundedCorner4]
+    (): IButtonStyles => concatStyleSets(copyLinkButtonStyles, themedCopyLinkButtonStyles(props.mobileView, theme)),
+    [props.mobileView, theme]
   );
 
-  if (props.mobileView) {
+  // for conditional compile
+  const mobileViewAddPeopleButton = (inviteLink?: string): JSX.Element => {
```

Comment:
nit: I'm very scared of local variables that are really potentially React components.

Can we extract these into file-private component `MobileViewAddPeopleButton` and use it as `<MobileViewAddPeopleButton .../>` below?